hack/lib/mesh_resources/authorization-policies/helm/eventing-e2e0.yaml

```diff
@@ -542,21 +542,3 @@ spec:
         - operation:
             ports:
               - "8022"
----
```

Comment:
hm that should already be in main: https://github.com/openshift-knative/serverless-operator/commit/bc47ce5976015cf8f7f2509a23727afe0c3f4cc8.